model-lab/utils/errorHandler.py

```
@@ -0,0 +1,63 @@
+logging.basicConfig(level=logging.INFO, format="%(levelname)s-%(message)s")
+
+def validate_email(email: str):
+    email_regex = r'^[\w\.-]+@[\w\.-]+\.\w+$'
+    if not re.match(email_regex, email):
+        raise HTTPException(
+        status_code=status.HTTP_422_UNPROCESSABLE_ENTITY,
+        detail="write your email correctly"
+    )
+
+def check_email_exists(email: str, db: Session):
+    existing_user = db.query(User).filter(User.email == email).first()
```

Ruff (0.14.5)

Undefined name `User`

(F821)